src/sdkPendle.ts

```diff
@@ -0,0 +1,71 @@
+import { ethers } from 'ethers';
+import { EvmAddress } from './types';
+import PendlePYLpOracleAbi from './abiPendle/PendlePYLpOracle.json';
+import PendleAdapter from './abiPendle/PendleAdapter.json';
```

Comment:
the `PendleAdapter` is part of the Cega contract. so that should still be part of the normal cega SDK

the `PendlePYLpOracleAbi` is part of the Pendle contract. so that should be part of the pendle SDK